pkg/cmd/cli/cmd/newbuild.go

```diff
@@ -109,6 +109,8 @@ func NewCmdNewBuild(fullName string, f *clientcmd.Factory, in io.Reader, out io.
 	cmd.Flags().StringVar(&config.ContextDir, "context-dir", "", "Context directory to be used for the build.")
 	cmd.Flags().BoolVar(&config.DryRun, "dry-run", false, "If true, do not actually create resources.")
 	cmd.Flags().BoolVar(&config.NoOutput, "no-output", false, "If true, the build output will not be pushed anywhere.")
+	cmd.Flags().StringVar(&config.SourceImage, "source-image", "", "Specify an image to use as source for the build.")
```

Comment:
Perhaps somewhere in the output of `oc new-build --help` we should say that `--source-image` and `--source-image-path` are meant to be used together?
This could be an usage example, or a brief mention in the description of `--source-image`.

I'm proud that users know us for the quality of the contextual help in our CLI :)